cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -126,17 +137,36 @@ public CgmesExportContext(Network network) {
             sshModelDescription.addDependencies(sshMetadata.getDependencies());
             sshModelDescription.setModelingAuthoritySet(sshMetadata.getModelingAuthoritySet());
         }
-        // TODO Fill EQ Model Description
-        addTopologicalNodeMappings(network);
+        addIidmMappings(network);
     }
 
+    /**
+     * @deprecated Not used anymore. To add the topological nodes mappings, use
+     * {@link CgmesExportContext#addIidmMappings(Network)} instead.
+     */
+    @Deprecated
     public void addTopologicalNodeMappings(Network network) {
-        // For a merging view we plan to call CgmesExportContext() and then addTopologicalNodesMapping(network) for every network
+        throw new ConversionException("Deprecated. Not used anymore");
```

Comment:
I'd rather not do that. If the method is to be kept as deprecated, the behaviour should emulate the previous one. If not possible, the method should be deleted as a breaking change. It is not good practice to have a method that reveals it doesn't work only in runtime... If it is kept, it is expected to work as before.

Reply:
Done as a breaking change